From 5726d1416c7d5e6741b73ec5940b0454711ea699 Mon Sep 17 00:00:00 2001
From: Robert Mader <robert.mader@collabora.com>
Date: Sat, 19 Oct 2024 00:25:03 +0200
Subject: [PATCH 08/13] ipa/simple: Add tuning file for IMX355

64 at 10 bits. The value was guessed from known values for similar
sensors and testing - on a Google Pixel 3a - suggest it's correct.

Adding this tuning file is partly motivated in order to serve as
example, as it's the first one for the simple IPA.

Signed-off-by: Robert Mader <robert.mader@collabora.com>
---
 src/ipa/simple/data/imx355.yaml | 11 +++++++++++
 src/ipa/simple/data/meson.build |  1 +
 2 files changed, 12 insertions(+)
 create mode 100644 src/ipa/simple/data/imx355.yaml

diff --git a/src/ipa/simple/data/imx355.yaml b/src/ipa/simple/data/imx355.yaml
new file mode 100644
index 000000000..f7d01b738
--- /dev/null
+++ b/src/ipa/simple/data/imx355.yaml
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: CC0-1.0
+%YAML 1.1
+---
+version: 1
+algorithms:
+  - BlackLevel:
+      blackLevel: 4096
+  - Awb:
+  - Lut:
+  - Agc:
+...
diff --git a/src/ipa/simple/data/meson.build b/src/ipa/simple/data/meson.build
index 92795ee4c..6e690f82d 100644
--- a/src/ipa/simple/data/meson.build
+++ b/src/ipa/simple/data/meson.build
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: CC0-1.0
 
 conf_files = files([
+    'imx355.yaml',
     'uncalibrated.yaml',
 ])
 
-- 
2.49.0

